java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -18,6 +18,8 @@ public interface GenericBaseCommands {
      * Removes the specified <code>keys</code> from the database. A key is ignored if it does not
      * exist.
      *
+     * @apiNote When in cluster mode, the command may route to multiple nodes when <code>keys</code>
+     *     map to different <code>hash slot</code>s.
```

Comment:
Why is "hash slot" in code tags? It's not the name of a parameter and it isn't a valid term name because it contains a space.